frontstage/templates/surveys/surveys-add.html

```diff
@@ -57,21 +57,21 @@ <h1 class="u-fs-xl ons-u-mt-s">Add a survey</h1>
                     "error": errorMsgText
                 })
             }}
-            <div class="ons-grid ons-grid--flex ons-grid--gutterless ons-grid--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
-                    <div class="ons-grid__col">
-                        {{
-                            onsButton({
-                                "text": "Continue",
-                                "type": "submit",
-                                "id": "continue_button",
-                                "submitType": "timer"
-                            })
-                        }}
-                    </div>
-                    <div class="ons-grid__col ons-u-ml-m">
-                        <a href="{{ url_for('surveys_bp.get_survey_list', tag='todo') }}" id="cancel_button" role="button">Cancel</a>
-                    </div>
+            <div class="ons-grid ons-grid-flex ons-grid--gutterless ons-grid-flex--vertical-center ons-grid--no-wrap@s ons-u-mt-m ons-u-mb-m">
+                <div class="ons-grid__col">
+                    {{
+                        onsButton({
+                            "text": "Continue",
+                            "type": "submit",
+                            "id": "continue_button",
+                            "submitType": "timer"
+                        })
+                    }}
                 </div>
+                <div class="ons-grid__col ons-u-ml-m">
```

Comment:
Not sure why this has been added

Reply:
To make things confusing, this looks like it has incorporated some of my changes for the add survey button. This would show as a changed when compared to the main branch.